app/classifier/tasks/generic-editor.cjsx

```diff
@@ -50,13 +50,21 @@ module.exports = React.createClass
           <small className="form-help">Add text and images for a window that pops up when volunteers click “Need some help?” You can use markdown to format this text and add images. The help text can be as long as you need, but you should try to keep it simple and avoid jargon.</small>
         </div>}
 
-      {if choicesKey?
+      {if choicesKey
         <div>
           <hr />
           <span className="form-label">Choices</span>
         </div>}
       {' '}
 
+    {if choicesKey is 'tools' and 'hide previous marks' in @props.project.experimental_tools # NOTE: assumes 'tools' is unique to drawing! --STI
+      [<label key="hide-previous-marks" className="pill-button">
```

Comment:
This array-of-one (and its `key` prop) isn't necessary (probably left over from a rebase, I think I remember working on this a few days ago).